### PR TITLE
Add suffixes to typos that start with "o"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -36817,24 +36956,77 @@ othewice->otherwise
...
+otpimizer->optimizer
+otpimizers->optimizers
+otpimizes->optimizes
+otpimizing->optimizing
 otpion->option
+otpional->optional
+otpionally->optionally
 otpions->options
 otput->output
+otside->outside
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:37020]: `onside`? Perhaps even `offside` and `topside`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -35971,10 +35984,17 @@ ocurrs->occurs
 ocurs->occurs
 odasee->odyssey
 odasees->odysseys
-oder->order, odor,
+oder->order, older, coder, odor, over, doer,
+odered->ordered
+odering->ordering
+oders->orders, coders, odors, doers,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:35990]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -36081,9 +36112,22 @@ oiginally->originally
 oiginals->originals
 oiginating->originating
 oigins->origins
-oints->points, pints,
+oint->point, pint, joint,
+ointed->pointed, jointed,
+ointer->pointer
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:36117]: This isn't common.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -36182,6 +36230,8 @@ ontaining->containing
 ontainor->container
 ontainors->containers
 ontains->contains
+ontent->content, intent,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:36233]: Is the suggestion `ontact->out-act`? I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:37020]: I couldn't find any of these typos.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -36081,9 +36112,22 @@ oiginally->originally
 oiginals->originals
 oiginating->originating
 oigins->origins
-oints->points, pints,
+oint->point, pint, joint,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:36115]: I'll add `lint` and its verbs. I couldn't find typos for the other ones.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**